docs/en/stack/ml/nlp/ml-nlp-classify-text.asciidoc

```diff
@@ -0,0 +1,135 @@
+[[ml-nlp-classify-text]]
+= Classify text
+
+:keywords: {ml-init}, {stack}, {nlp}, {lang-ident}, text classification, \
+zero-shot text classification
+
+:description: NLP tasks that classify input text into categories or determine \
+the language of text. 
+
+These NLP tasks enable you to identify the language of text and categorize 
+unstructured input text:
+
+* <<ml-nlp-lang-ident>>
+* <<ml-nlp-text-classification>>
+* <<ml-nlp-zero-shot>>
+
+include::ml-nlp-lang-ident.asciidoc[]
+
+
+[discrete]
+[[ml-nlp-text-classification]]
+== Text classification
+
+Text classification assigns the input text to one of multiple classes or 
+categories that best describe the text. The categories used depend 
```

Comment:
```suggestion
Text classification assigns the input text to one of multiple classes
 that best describe the text. The classes used depend 
```
maybe "labels"? instead of "classes"?